tests/app/broadcast_message/test_rest.py

```diff
@@ -489,13 +489,20 @@ def test_update_broadcast_message_status_doesnt_let_you_update_other_things(admi
     }]
 
 
+@pytest.mark.parametrize('user_is_platform_admin', [True, False])
 def test_update_broadcast_message_status_stores_cancelled_by_and_cancelled_at(
```

Comment:
Optional: since this test is now checking two things (the permission check is implicit in the test still passes), we could rename it to make this more explicit e.g. `test_update_broadcast_message_allows_service_user_and_platform_admin_to_cancel`.

Reply:
Good idea 👍🏼